ladybug_comfort/solarcal.py

```diff
@@ -21,18 +21,19 @@
 from ladybug.skymodel import calc_sky_temperature
 from ladybug.futil import csv_to_num_matrix
 
-from os.path import dirname, abspath
-import inspect
+from os.path import dirname, join
```

Comment:
not sure how I feel about overwriting the python `join`. I would import os and fully type the commands but up to you.